app/controllers/repos_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ReposController < ApplicationController
+  def index
+    github_service = GithubService.new(current_user.token)
+    @repos = github_service.find_repos
```

Comment:
@jmejia that makes sense, thank you. where do i start with testing for that? since in my features, i am accessing the same information i would through objects (i think)